src/common/schemas/output/ledger-closed.json

```diff
@@ -4,15 +4,48 @@
   "description": "A ledgerClosed event message",
   "type": "object",
   "properties": {
-    "feeBase": {"type": "integer", "minimum": 0},
-    "feeReference": {"type": "integer", "minimum": 0},
-    "ledgerHash": {"$ref": "hash256"},
-    "ledgerVersion": {"$ref": "ledgerVersion"},
-    "ledgerTimestamp": {"type": "string", "format": "date-time"},
-    "reserveBase": {"type": "integer", "minimum": 0},
-    "reserveIncrement": {"type": "integer", "minimum": 0},
-    "transactionCount": {"type": "integer", "minimum": 0},
-    "validatedLedgerVersions": {"type": "string"}
+    "feeBase": {
+      "type": "integer",
+      "minimum": 0,
+      "description": "Base fee, in drops."
+    },
+    "feeReference": {
+      "type": "integer",
+      "minimum": 0,
+      "description": "Cost of the 'reference transaction' in 'fee units'."
```

Comment:
This is also confusing, although sadly, I don't think we have a good explanation that's not overly complicated (because the product itself is just complicated here). Maybe not worth exposing in RippleAPI?